setup.cfg

```diff
@@ -17,7 +17,7 @@ python_requires = >= 3.7
 [options.extras_require]
 dev =
   # for tests
-  matrix-synapse
+  matrix-synapse @ git+https://github.com/matrix-org/synapse.git@rei/modapi_acc_data
```

Comment:
Should we be specifying a minimum bound of Synapse somewhere?

Reply:
Good point, I'll add that in

Reply:
Is there a reason we're specifying the version bound as an optional dev-only extra rather than as a hard requirement on the package?

Reply:
I think this version bound is purely so tests pass (since I assume the module relies on changes that weren't already merged into develop or released in a stable version of Synapse at the time).